.jshintignore

```diff
@@ -9,6 +9,7 @@ external/jpgjs/
 external/jasmine/
 external/cmapscompress/
 external/importL10n/
+external/builder/fixtures2/
```

Comment:
I'm not really liking the name `fixtures2` as it does not seem very descriptive. Can we change it to, for example, `fixtures_esprima` or something like that to make it more clear that it is about the Esprima preprocessor and not our own?